src/block_service/fds/fds_service.cpp

```diff
@@ -549,18 +522,15 @@ error_code fds_file_object::get_file_meta()
         _md5sum = iter->second;
 
         _has_meta_synced = true;
-        return ERR_OK;
     } catch (const galaxy::fds::GalaxyFDSClientException &ex) {
         if (ex.code() == Poco::Net::HTTPResponse::HTTP_NOT_FOUND) {
-            return ERR_OBJECT_NOT_FOUND;
+            err = ERR_OBJECT_NOT_FOUND;
         } else {
-            derror_f("fds getObjectMetadata failed: parameter({}), code({}), msg({})",
-                     _name.c_str(),
-                     ex.code(),
-                     ex.what());
```

Comment:
I suggest reserving the error log here, `FDS_EXCEPTION_HANDLE` will also set err as `ERR_FS_INTERNAL`.